src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Header from './components/header';
+import Main from "./components/main";
+import Improve from "./components/improve";
+import Relax from "./components/relax";
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+
+function App() {
```

Comment:
please fix formatting.

Reply:
What is wrong with the formatting her? Did I miss something, or maybe imports are incorrect?

Reply:
some content of `return` is moved too far.